runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the container process by the runtime.
+The runtime may leak additional file descriptors into the container process to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
```

Comment:
http://man7.org/linux/man-pages/man7/pid_namespaces.7.html calls it as the "init" process for the pid namespace. However, we could be in a situation where we don't use a new pid namespace.